tools/pipelines/templates/include-test-real-service.yml

```
@@ -340,12 +344,43 @@ stages:
               key: '"compat-version-installs" | "$(Agent.OS)" | "${{ parameters.testCommand }}" | "${{ variant.name }}"'
               path: ${{ parameters.testWorkspace }}/node_modules/@fluid-private/test-version-utils/node_modules/.legacy/
 
+        # Only check out tenants from the TRIPS tenant pool if we are running tests against ODSP
```

```suggestion
        # Only check out tenants from the tenant pool if we are running tests against ODSP
```

probably no need to mention the service name in OSS. apply similar suggestion elsewhere throughout please :)

---

eh it's fine